rastervision/data/label_store/semantic_segmentation_raster_store_config.py

```diff
@@ -64,27 +92,67 @@ def update_for_command(self,
                 msg = 'No URI set for SemanticSegmentationRasterStoreConfig'
                 io_def.add_missing(msg)
 
+            for vo in self.vector_output:
+                io_def.add_input(vo['uri'])
+
         return io_def
 
 
 class SemanticSegmentationRasterStoreConfigBuilder(LabelStoreConfigBuilder):
     def __init__(self, prev=None):
         config = {}
         if prev:
-            config = {'uri': prev.uri, 'rgb': prev.rgb}
+            config = {
+                'uri': prev.uri,
+                'vector_output': prev.vector_output,
+                'rgb': prev.rgb,
+            }
 
         super().__init__(SemanticSegmentationRasterStoreConfig, config)
 
     def from_proto(self, msg):
-        return self.with_uri(msg.semantic_segmentation_raster_store.uri) \
-                   .with_rgb(msg.semantic_segmentation_raster_store.rgb)
+        uri = msg.semantic_segmentation_raster_store.uri
+        rgb = msg.semantic_segmentation_raster_store.rgb
+        vo = msg.semantic_segmentation_raster_store.vector_output
+
+        return self.with_uri(uri) \
+                   .with_vector_output(vo) \
+                   .with_rgb(rgb)
 
     def with_uri(self, uri):
         """Set URI for a GeoTIFF used to read/write predictions."""
         b = deepcopy(self)
         b.config['uri'] = uri
         return b
 
+    def with_vector_output(self, msg):
+        """Configure vector output for predictions.
+
+            Args:
+                msg: Either a list of dictionaries or a protobuf
```

Comment:
I'm pretty sure that none of the other `with` methods take protobuf messages as input (with the exception of with_class_map). I guess it doesn't hurt, but it doesn't seem very useful. I also think the name `config` is better than `msg` since it might be a dict . Also, I would add something to the validate method to check that the value of mode is valid.